### PR TITLE
Exposing Campaign editor description field labels

### PR DESCRIPTION
**Pull request by jkueloc**
This request is for ticket #1266

### CODE REVIEW

**Review by rabiloc — APPROVED** [review]
Looks good

**Comment by rabiloc** [comment]
merged

File: concordia/static/admin/editor-preview.js [hunk 1]
```diff
@@ -20,7 +20,7 @@
             .parents('.form-row')
             .first();
         $formRow.addClass('codemirror-with-preview');
-        $formRow.find('label').remove();
+        //$formRow.find('label').remove();
```
Comment (acdha) [inline concordia/static/admin/editor-preview.js:23]: If we're changing this we should delete the line so there's no question about intent

Comment (jkueloc) [reply concordia/static/admin/editor-preview.js:23]: I agree. I'll make the change.

**Pull request merged**